Documentation/Signoffs/MasterMCU.md

```diff
@@ -0,0 +1,81 @@
+MASTER MICROCONTROLLER
+------------------------
+### Function of the Subsystem
+The master microcontroller (MMCU) must receive data from four seperate local microcontrollers that interface with the
+sensors in other subsystems. This MMCU will read serial data from each local MCU and then send that data to the PC to
+be processed in the simulation.
+
+### Specifications & Constraints
+- Must connect to no less than four local microcontrollers
+- Must provide real-time data transfer of under 100 milliseconds
+- Shall connect to the PC running the simulation
+- Shall have sufficient GPIO pins for communication protocols
+- Shall be powered by the PC
+- Code must not cause deadlock or inhibit any device(s) from sending/receiving data
+
+The definition of real-time is subjective and therefore can not be directly defined. In the case of this system, real-time was
+based on the average human reaction time of 250 milliseconds [1]. Given that delay being considerably high for a real-time system, 
```

Comment:
This makes sense. But I don't know if that reference is appropriate. That is reaction time, not time to perceive. And this assumes this microcontroller is the only thing causing delay. It's not. Every link in your dataflow chain will cause prop delay. 

I would look into how large of a lag in a video game or similar scenario can be detected by people.